lib/matplotlib/backends/backend_pdf.py

```diff
@@ -1527,8 +1527,7 @@ def writeXref(self):
         borken = False
         for offset, generation, name in self.xrefTable:
             if offset is None:
-                print('No offset for object %d (%s)' % (i, name),
-                      file=sys.stderr)
+                _log.error('No offset for object %d (%s)', i, name)
```

Comment:
Just lift the AssertionError that gets raised a few lines below to this point, instead.